### PR TITLE
Switch to ghcr.io

### PR DESCRIPTION
**Pull request by ikifar2012**
Switch repo to GitHub container registry

### CODE REVIEW

File: homeassistant-supervised/DEBIAN/postinst [hunk 1]
```diff
@@ -127,3 +127,6 @@ systemctl start hassio-supervisor.service
 info "Installing the 'ha' cli"
 chmod a+x "${PREFIX}/bin/ha"
 
+info "Installation Complete!"
+info "Within a few minutes you will be able to reach Home Assistant at:"
+info "homeassistant.local:8123 or your machine's ip address:8123"
```
Comment (frenck) [inline homeassistant-supervised/DEBIAN/postinst:132]:
```suggestion
info "Within a few minutes you will be able to reach Home Assistant at:"
info "http://homeassistant.local:8123 or using the IP address of your"
info "machine: http://IP_ADDRESS:8123"
```

Ideally, we should get the IP address for the primary interface and show it in the message.

But, that (+ this message) may be a bit of scope for this PR.

Comment (ikifar2012) [reply homeassistant-supervised/DEBIAN/postinst]: Moved to #190

**Comment by ikifar2012** [comment]
Everything good to merge?

**Review by frenck — APPROVED** [review]
Thanks, @ikifar2012 👍

**Pull request merged**